automation/build-keycloak.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh -e
```

Comment:
Please integration into build-srpm.sh provided in https://github.com/oVirt/ovirt-engine-keycloak/pull/4